backend/db/migrations/20200709112529-addExternalEvents.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    const isPostgres = queryInterface.sequelize.options.dialect === 'postgres'
+
+    return queryInterface.createTable('external_events', {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      shop_id: {
+        type: Sequelize.INTEGER,
+        references: { model: 'shops', key: 'id' }
+      },
+      service: Sequelize.STRING,
```

Comment:
I think it would be nice to use enums rather than strings.
It makes it easier to understand the schema.
Something we discussed in Tom's productivity doc and earlier this week during the offsite as part of the "best coding practices" we would like to apply moving forward.

Reply:
Will change it, thanks